modules/run-consul/run-consul

```diff
@@ -43,6 +43,7 @@ function print_usage {
   echo -e "  --ca-path\t\tPath to the directory of CA files used to verify outgoing connections. Optional. Must be specified with --enable-rpc-encryption."
   echo -e "  --cert-file-path\t\tPath to the certificate file used to verify incoming connections. Optional. Must be specified with --enable-rpc-encryption and --key-file-path."
   echo -e "  --key-file-path\t\tPath to the certificate key used to verify incoming connections. Optional. Must be specified with --enable-rpc-encryption and --cert-file-path."
+  echo -e "  --environment\t\tA list of key/value pairs in the form 'KEY=\"val\",KEY2=\"val2\"' to pass to Consul as environment variables when starting it up. Optional."
```

Comment:
The typical approach with bash is to pass lists by specifying the same option multiple times:

```
run-consul --env FOO=abc --env BAR=def
```

I suppose yours works with careful quoting too, but it strikes me as more error prone:

```
run-consul --environment "FOO=abc BAR=def"
```

Reply:
Sure. I wanted to keep it verbatim to pass to the config file directly.